.github/workflows/migrations.yml

```diff
@@ -11,7 +11,7 @@ on:
     branches: [ main ]
 
 env:
-  MONGODB_VERSION: 5.0
+  MONGODB_VERSION: 6.0.15
```

Comment:
Is not only 6.0 the right version to set, we don't want to update the workflow for each patch version update

Reply:
agree